reference-architecture/osp-dns/deploy-dns.yaml

```diff
@@ -42,7 +42,8 @@
     register: stack_output_raw
 
   - set_fact:
-      stack_output: "{{ stack_output_raw.stdout|from_json }}"
+      # Need "safe" to create an Ansible safe dictionary object.
+      stack_output: "{{ stack_output_raw.stdout|safe }}"
```

Comment:
One question: is it possible to only 'safe' the particular output piece you need, when you need it? I'm just worried that this change could have unintended consequences when other parts of the playbook read this data.

Reply:
It may be possible to pull apart that object but as it is now it will NOT complete successfully on the latest version of ansible 2.4.2 that openshift-ansible requires. I believe this is some old code that may have worked in the past but does not work now. Please feel free to run it yourself to verify.

Reply:
Ah, okay - looks good then!